libs/commands/version/README.md

```diff
@@ -551,6 +552,23 @@ lerna version prepatch --preid next
 When run with this flag, `lerna version` will increment `premajor`, `preminor`, `prepatch`, or `prerelease` semver
 bumps using the specified [prerelease identifier](http://semver.org/#spec-item-9).
 
+
+### `--premajor-version-bump`
+
+This option allows you to control how lerna handles bumping versions for `premajor` version release packages. 
+By default, this is configured as `"semver"`, and any `premajor` package is bumped as `minor`
```

Comment:
The "any premajor package is bumped as minor" is not true. Depending on the changelog-preset, fixes will still be bumped as patches. It's just that non-breaking features trigger minor bumps.

```suggestion
By default, this is configured as `"semver"`. This option only affects non-breaking changes in `premajor` packages. Breaking changes in `premajor` packages will always trigger a `minor` bump.
```